stdlib/public/Darwin/Foundation/Data.swift

```diff
@@ -41,7 +41,7 @@ internal func __NSDataIsCompact(_ data: NSData) -> Bool {
 #else
 
 @_exported import Foundation // Clang module
-import _SwiftFoundationOverlayShims
+@_implementationOnly import _SwiftFoundationOverlayShims
 import _SwiftCoreFoundationOverlayShims
```

Comment:
shouldn't this also be `@_implementationOnly` too?

Reply:
It indeed should! Fixed.